modules/SofaBoundaryCondition/LinearMovementConstraint.inl

```diff
@@ -248,24 +249,15 @@ template <class MyCoord>
 void LinearMovementConstraint<DataTypes>::interpolatePosition(Real cT, typename std::enable_if<!std::is_same<MyCoord, defaulttype::RigidCoord<3, Real> >::value, VecCoord>::type& x)
 {
```

Comment:
From SOFA-dev meeting: shouldn't this function handle `d_relativeMovements` too?

Reply:
ping @raphaeldeimel :-)

Reply:
you're totally right ;)